### PR TITLE
Decrease number of CI invocations for Elasticsearch

### PR DESCRIPTION
**Pull request by MiguelWeezardo**

### CODE REVIEW

**Review by kokosing — APPROVED** [review]

File: plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/TestElasticsearchBackpressure.java [hunk 1]
```diff
@@ -63,14 +68,17 @@ public final void destroy()
...
+
         // Check that JMX stats show no sign of backpressure
-        assertQueryReturnsEmptyResult("SELECT 1 FROM jmx.current.\"trino.plugin.elasticsearch.client:*\" WHERE \"backpressurestats.alltime.count\" > 0");
-        assertQueryReturnsEmptyResult("SELECT 1 FROM jmx.current.\"trino.plugin.elasticsearch.client:*\" WHERE \"backpressurestats.alltime.max\" > 0");
+        assertThat(stat.getAllTime().getCount()).as("Backpressure all-time count stats should be 0").isEqualTo(0);
+        assertThat(stat.getAllTime().getMax()).as("Backpressure all-time max stats should be 0").isEqualTo(0);
 
         assertQuerySucceeds("SELECT * FROM orders");
 
         // Check that JMX stats show requests have been retried due to backpressure
```
Comment (findepi) [inline plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/TestElasticsearchBackpressure.java:80]: is "JMX" still applicable here

File: plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/TestElasticsearchBackpressure.java [hunk 2]
```diff
@@ -63,14 +68,17 @@ public final void destroy()
     @Test
     public void testQueryWithBackpressure()
     {
+        ElasticsearchClient client = ((DistributedQueryRunner) getQueryRunner()).getCoordinator().getInstance(Key.get(ElasticsearchClient.class));
+        TimeStat stat = client.getBackpressureStats();
+
         // Check that JMX stats show no sign of backpressure
```
Comment (findepi) [inline plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/TestElasticsearchBackpressure.java:74]: is "JMX" still applicable here

**Comment by MiguelWeezardo** [comment]
I'm getting `[Guice/JitDisabled]: Explicit bindings are required and ElasticsearchClient is not explicitly bound`. I assumed the class would be bound by `ElasticsearchConnectorFactory::create()` which loads `ElasticsearchConnectorModule`.

File: plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/TestElasticsearchBackpressure.java (hunk 1 above)
Comment (MiguelWeezardo) [reply plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/TestElasticsearchBackpressure.java]: No, removed

**Comment by colebow** [comment]
:wave: @MiguelWeezardo - this PR is inactive and doesn't seem to be under development. If you'd like to continue work on this at any point in the future, feel free to re-open.

**Pull request closed without merging**